security/certificate_types_descriptions/service-ca-certificates.adoc

```diff
@@ -27,6 +27,11 @@ The service CA expiration of 26 months is longer than the expected upgrade inter
 A manually-rotated service CA does not maintain trust with the previous service CA. You might experience a temporary service disruption until the pods in the cluster are restarted, which ensures that pods are using service serving certificates issued by the new service CA.
 ====
 
+[IMPORTANT]
+====
+Applications using the `service-ca` certificate must use a truststore capable of dynamically accepting new CA certificates. Otherwise, when automated rotation occurs, the application pods will require a restart.
```

Comment:
```suggestion
Applications using the `service-ca` certificate must be capable of dynamically reloading CA certificates. Otherwise, when automated rotation occurs, the application pods might require a restart in order to rebuild certificate trust.
```

Reply:
Done.